src/components/Landing/Hero.tsx

```diff
@@ -1,36 +1,127 @@
+import { useState, useEffect } from "react";
 import {
   Container,
-  Text,
   Title,
-  Image,
-  Button,
   useMantineTheme,
+  BackgroundImage,
+  Image,
 } from "@mantine/core";
 import styles from "./Hero.module.css";
 
+const images = [
+  {
+    src: "/images/hero/photo1.jpg",
+    title: "The life moments",
+    text: "Learn the art of analog photography",
+  },
+  {
+    src: "/images/hero/photo2.jpg",
+    title: "Timeless Shots",
+    text: "Discover the beauty of film photography",
+  },
+  {
+    src: "/images/hero/photo3.jpg",
+    title: "Retro Elegance",
+    text: "Master manual settings and composition",
+  },
+  {
+    src: "/images/hero/photo4.jpg",
+    title: "Unique Aesthetics",
+    text: "Create unique images with real grain",
+  },
+  {
+    src: "/images/hero/photo6.jpg",
+    title: "Keep Memories",
+    text: "Every photo tells a story",
+  },
+];
+
 export default function Hero() {
   const theme = useMantineTheme();
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [fade, setFade] = useState(true);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setFade(false);
+      setTimeout(() => {
+        setCurrentIndex((prevIndex) => (prevIndex + 1) % images.length);
+        setFade(true);
+      }, 300);
+    }, 5000);
+
+    return () => clearInterval(interval);
+  }, []);
+
+  const handlePrev = () => {
+    setFade(false);
+    setTimeout(() => {
+      setCurrentIndex(
+        (prevIndex) => (prevIndex - 1 + images.length) % images.length,
+      );
+      setFade(true);
+    }, 300);
+  };
+
+  const handleNext = () => {
+    setFade(false);
+    setTimeout(() => {
+      setCurrentIndex((prevIndex) => (prevIndex + 1) % images.length);
+      setFade(true);
+    }, 300);
+  };
+
   return (
     <div className={styles.hero}>
```

Comment:
again
Box an sx 
if not possible, only then resort to css
we want to writeclean and reusable components so that we have less work in the future